### PR TITLE
Add pyramid's example

### PR DESCRIPTION
**Pull request by podhmo**
Hello. This is a example for pyramid.

### CODE REVIEW

**Comment by podhmo** [comment]
if TODO comments are a bit messy, I will remove it.

File: examples/pyramid_example.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
+"""A simple number and datetime addition JSON API.
+Run the app:
+
+    $ python examples/flask_example.py
```
Comment (sloria) [inline examples/pyramid_example.py]: This needs to be changed to `python examples/pyramid_example.py`

File: examples/pyramid_example.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+from webargs import fields, validate
+from webargs.pyramidparser import use_args, use_kwargs
+
+
+hello_args = {
+    'name': fields.Str(missing='Friend')
+}
+
+
+# TODO: support
```
Comment (sloria) [inline examples/pyramid_example.py]: Can you remove these and report issues for the missing features you are referring to?

**Review by sloria — CHANGES_REQUESTED** [review]
Looks good! Just a few tiny changes and this is good to merge.

File: examples/pyramid_example.py (hunk 2 above)
Comment (podhmo) [reply examples/pyramid_example.py]: OK. I will fix.

**Comment by podhmo** [comment]
fixed (and squashed).

**Review by sloria — APPROVED** [review]

**Comment by sloria** [comment]
Thanks!

**Pull request merged**